Automattic/wp-calypso
# Copy Site: Prefetch data for site actions

#### Proposed Changes

'Copy Site' and 'Manage Plugins' appear instantly when applicable, instead of after a delay. (Prefetch data)

NOTE:

Has changes that already esists in https://github.com/Automattic/wp-calypso/pull/72291.

I'll make sure that I'll update this pr when this is merged.

#### Testing Instructions

1. Open the dev console.
2. Make sure you have a  site with `Business` plan and atomic.
3. Navigate to `/sites`
4. Ensure that the requests you see, exist:
        - Only one request to `me/purchases`
        - `/sites/<site id>/features` requests only for the sites you see on your viewport.
5. Make sure that you find the site in your list. Click on the three dots. `Copy Site` menu and `Manage Plugin` menu should be there.
![prefetch-sites-menu](https://user-images.githubusercontent.com/497103/216055038-33611da0-8635-4581-b282-17ef4217b278.png)


#### Pre-merge Checklist





Fixes https://github.com/Automattic/wp-calypso/issues/72275
